### ISSUE TITLE
ENHANCEMENT: add/edit background/UI

### ISSUE THREAD

**Issue by jsooter1** [advplyr/audiobookshelf#186]
I would love to have an option to choose different backgrounds/wood types. I did a google image search for "wood boards background" and was overwhelmed with all of the potential options.

Also, have you see Google's infinite bookshelf (WebGL Bookcase)? Pretty cool UI.
https://bookcase.chromeexperiments.com/

**Comment by p1xelll** [comment]
Background change could be for series and collections. 
![series](https://user-images.githubusercontent.com/81173399/142047764-555675f4-0176-4977-9e43-a6763f4d4188.png)

The UI is interesting but the UX is terrible. It can lag on a slower pc.

**Comment by advplyr** [comment]
The infinite bookshelf is pretty cool. I see things like that being plugins/extensions in the future.

 I think different wood backgrounds would be fairly easy to implement and wouldn't add any additional overhead. Can you provide some options? They need to be repeating horizontally.

For the book landing pages, I like the way Plex handles that with the half faded images. The only thing is for books the images are usually not big posters so it doesn't look as good.

**Comment by advplyr** [comment]
I decided to play around with this a bit and added a "Texture" button to the bottom right of the bookshelf if you have experimental features on.
I added some example textures that can be selected. They aren't persisted.

![image](https://user-images.githubusercontent.com/67830747/143726620-e672e08b-f8f2-4b41-a279-e310158e3c60.png)

There will also need to be an option for changing the shelf color, or a shelf color will come with the texture.
There are a lot of good looking free wood textures, but I didn't see many that are seamless/repeating.
This is in `v1.6.30`
